documentation/src/main/resources/_posts/2023-08-23-ditto-benchmark.md

```diff
@@ -0,0 +1,465 @@
+---
+title: "Eclipse Ditto Benchmark"
+published: false
+permalink: 2023-08-23-ditto-benchmark.html
```

Comment:
Hi @vvasilevbosch 

`published` needs to be changed to `true` and the blog date has to be updated for both `permalink` and in the filename to the publish-date (probably next week?).